src/main/java/picard/vcf/CallingMetricAccumulator.java

```diff
@@ -133,7 +133,7 @@ public void accumulate(final VariantContext vc) {
      * @return Sample name if there is only one sample that contains alternate allele(s), else null if either multiple samples that
      * are not homref, or no samples that are not homref.
      */
-    protected static String getSingletonSample(final VariantContext vc) {
+    public static String getSingletonSample(final VariantContext vc) {
 
         // peek can only change effectively final variables...workaround
         final String[] sampleName = new String[1];
```

Comment:
I know this isn't part of the PR, but I think line 150 here is wrong as if there's a single sample that is homozygous variant, then its allele count will be 2 which obviously does not equal 0 so we will not return that sample as a singleton